### PR TITLE
Feature/demo page

### PR DESCRIPTION
**Pull request by aabounegm**
Add a page for showcasing the components. Didn't design anything properly, so it is just a proof-of-concept. Wrote a GitHub Action to deploy that demo page to GitHub Pages, but cannot test it without first making a release.

### CODE REVIEW

File: demo/App.svelte [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<script>
+	import {
```
Comment (illright) [inline demo/App.svelte:2]: Tabs

File: jsconfig.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+	"compilerOptions": {
+		"baseUrl": "./",
+		"paths": {
+			"@/*": ["src/*"],
```
Comment (illright) [inline jsconfig.json:5]: You sure this is necessary? It was introduced to Innopoints by Max and I never really saw the need in it, it's confusing

File: package.json [hunk 3]
```diff
@@ -8,25 +8,33 @@
...
   "keywords": [
     "svelte"
   ],
   "files": [
     "src",
     "dist"
-  ]
+  ],
+  "dependencies": {
+    "sirv-cli": "^0.4.5"
```
Comment (illright) [inline package.json:38]: That definitely should not be a dependency

File: jsconfig.json (hunk 2 above)
Comment (aabounegm) [reply jsconfig.json:5]: It helps avoid writing a lot of '../../' imports. Also, this particular file does nothing but tell VS Code about that alias. The actual alias is defined in the rollup config

Comment (illright) [reply jsconfig.json:5]: But in this case it only substitutes `@/` for `src/`, why couldn't we just write `src/` directly?

**Pull request merged**